### DISCUSSION TITLE
Unbacom - Commander rework

### DISCUSSION THREAD

**Discussion by Jazcash** [beyond-all-reason/rfcs#2] [General]
Discussion Thread for "Unbacom - Commander rework"

## Links
[RFC PR](https://github.com/beyond-all-reason/rfcs/pull/1)
[Full Rendered Proposal](https://github.com/Avaristimo/rfcs/blob/master/active-rfcs/0001-unbacom.md)

**Comment by WatchTheFort** [comment]
If Commanders earn XP by dealing damage, doesn't this put the Armada Commander at a disadvantage because the EMP doesn't inflict damage?
